src/aat/resources/features/F-067/S-212_Get_Case_Data.td.json

```diff
@@ -0,0 +1,75 @@
+{
+  "_guid_": "S-212_Get_Case_Data",
+  "title": "to get the details about case event for the case just created",
+  "productName": "CCD Data Store",
+  "operationName": "to get the details about case event for the case just created",
```

Comment:
operationName not correct

Reply:
updated by "Retrieve a CaseView Event by case and event id for dynamic display"

Reply:
sorry missed this one now done

Reply:
done